tests/importer/importer.cpp

```diff
@@ -501,7 +501,7 @@ TEST(Importer, library)
     EXPECT_EQ(nullptr, importer->library(999));
 }
 
-TEST(Importer, tryingStuffOut)
+TEST(Importer, rangeOfValidSituations)
```

Comment:
Hmm... that name still doesn't tell me what this test is about, but I guess it's still better than `tryingStuffOut`...

Reply:
What would you suggest?

Reply:
Long test name: `resolveImportsUsingNeededAndUnneededModelsWithDifferentTypesOfKeys`? :)

Reply:
Haha, nope :)

Reply:
At least, I would have known what the test was about, but whatever.